message/router.go

```diff
@@ -30,6 +30,9 @@ var (
 // (because msg.Ack() was sent in HandlerFunc or Subscriber supports multiple consumers).
 type HandlerFunc func(msg *Message) ([]*Message, error)
 
+// NoPublishHandlerFunc is HandlerFunc alternative, which doesn't produce any message.
+type NoPublishHandlerFunc func(msg *Message) error
```

Comment:
How about `ReadOnlyHandlerFunc` or `ConsumeOnlyHandlerFunc`?

Reply:
sounds better @maclav3 ? and does we want to also change AddNoPublisherHandler?